src/components/controls/MediaStreamControl.tsx

```diff
@@ -24,14 +48,38 @@ export class MediaStreamControl extends React.PureComponent<IMediaControlProps,
         )
     }
 
+    private getAudioEnabled(): boolean {
+        const track = this.props.stream.getAudioTracks()[0];
+        return track ? track.enabled : false;
+    }
+
+    private setAudioEnabled(enabled: boolean): void {
+        const track = this.props.stream.getAudioTracks()[0];
+        if (track) {
+            track.enabled = enabled;
+        }
+    }
+
+    private getVideoEnabled(): boolean {
+        const track = this.props.stream.getVideoTracks()[0];
+        return track ? track.enabled : false;
+    }
+
+    private setVideoEnabled(enabled: boolean): void {
+        const track = this.props.stream.getVideoTracks()[0];
+        if (track) {
+            track.enabled = enabled;
```

Comment:
Use `forceUpdate` here to trigger render and keep a single source of truth.

```tsx
if (track) {
    track.enabled = enabled;
    this.forceUpdate()
}
```

Reply:
Done.